examples/node/app.js

```diff
@@ -394,4 +399,4 @@ function fixWidth(str, len) {
     return str;
 }
 
-matrixClient.startClient(numMessagesToShow); // messages for each room.
+matrixClient.startClient({ initialSyncLimit: numMessagesToShow }); // messages for each room.
```

Comment:
not a new problem, but this comment is... opaque.

```suggestion
matrixClient.startClient({ initialSyncLimit: numMessagesToShow });
```